### PR TITLE
Task #197014 refactor: Get State, District, Block, Village dependancies API

### PR DESCRIPTION
**Pull request by manojLondhe**

### CODE REVIEW

File: src/src/modules/geolocation/geolocation.controller.ts [hunk 1]
```diff
@@ -1,17 +1,17 @@
-import { Controller, Get, Param, UseInterceptors } from '@nestjs/common';
-import { GeolocationService } from './geolocation.service';
-import { SentryInterceptor } from 'src/common/interceptors/sentry.interceptor';
+import { Controller, Get, Param, UseInterceptors } from "@nestjs/common";
+import { SentryInterceptor } from "src/common/interceptors/sentry.interceptor";
+import { GeolocationService } from "./geolocation.service";
```
Comment (Tusharmahajan12) [inline src/src/modules/geolocation/geolocation.controller.ts:3]: why change this sequence.

File: src/src/modules/geolocation/geolocation.service.ts [hunk 2]
```diff
@@ -1,154 +1,151 @@
...
-            'x-hasura-admin-secret': process.env.HASURA_ADMIN_SECRET,
-            'Content-Type': 'application/json',
-          },
-        })
-        .pipe(map((res) => res.data)),
-    );
-  }
+	constructor(private readonly hasuraService: HasuraService) {}
+
+	public async findAll(tableName: string, filters: Object = {}) {
```
Comment (Tusharmahajan12) [inline src/src/modules/geolocation/geolocation.service.ts:8]: Tab space count change 4 to 2 then format the file.

Comment (manojLondhe) [reply src/src/modules/geolocation/geolocation.service.ts:8]: it is as per
/src/.prettierrc.json

File: src/src/modules/geolocation/geolocation.controller.ts (hunk 1 above)
Comment (manojLondhe) [reply src/src/modules/geolocation/geolocation.controller.ts:3]: done via command -> typescript -> sort imports

**Pull request merged**